exercises/forth/cases_test.go

```diff
@@ -0,0 +1,223 @@
+package forth
+
+// Source: exercism/x-common
+// Commit: 8804a76 Remove non-ASCII characters tests from forth exercise
+
+var testCases = []testCase{
+	// parsing and numbers
+	{
+		"empty input results in empty stack",
+		[]string{},
+		[]int{},
+	},
+	{
+		"numbers just get pushed onto the stack",
+		[]string{"1 2 3 4 5"},
+		[]int{1, 2, 3, 4, 5},
+	},
+	{
+		"all non-word characters are separators",
+		[]string{"1\x002\x133\n4\r5 6\t7"},
+		[]int{1, 2, 3, 4, 5, 6, 7},
+	},
+
+	// addition
+	{
+		"can add two numbers",
+		[]string{"1 2 +"},
+		[]int{3},
+	},
+	{
+		"errors if there is nothing on the stack",
```

Comment:
it's a little interesting because there are many tests whose description is "errors if there is nothing on the stack", and it may be hard to tell the difference if I get in my output that one of them has failed. It would make sense if we prepend the section name, making it "addition errors if there is nothing on the stack", but prepending the section name may not make sense for all sections (or maybe it does; I haven't checked all of them). Anyone have a good solution here?